tools/prove_wrapper

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -euo pipefail
+
+OUTPUT=$(mktemp)
+
+echo "Running prove with TAP output check ..."
```

Comment:
small think but could we add some distinctive mark in the beginning of the string?

Reply:
Do you have any suggestion?

Reply:
Maybe `... Running prove with TAP output check` or `[CUSTOM_PROVE] Running prove with TAP output check` or something like that. but I guess you can merge either way for now as is and think about that (if that matters later)